File: Data Structures & Algorithms/duplicate-integer/submission-5.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        int s=nums.size();
        int c=0;
        sort(nums.begin(),nums.end());
        for(int i=0;i<s;i++){
            for(int j=i+1;j<s;j++){
                if(nums[i]==nums[j]){
                    return true;
                }
            }
        }
        return false;
    }
};

File: Data Structures & Algorithms/majority-element/submission-5.java
class Solution {
    public int majorityElement(int[] nums) {
        int maxel=0;
        int maxfreq=0;
        HashMap<Integer,Integer>map=new HashMap<>();
        for(int i=0;i<nums.length;i++){
            map.put(nums[i],map.getOrDefault(nums[i],0)+1);
            if(map.get(nums[i])>maxfreq){
                maxfreq=map.get(nums[i]);
                maxel=nums[i];
            }
        }
        return maxel;
    }
}